KnowSheet/scripts/full-test.sh

```diff
@@ -11,16 +11,21 @@ set -u -e
 CPPFLAGS="-std=c++11 -g -Wall -W -fprofile-arcs -ftest-coverage -DBRICKS_COVERAGE_REPORT_MODE"
 LDFLAGS="-pthread"
 
-FULLTESTDIR=zzz_full_test  # The `zzz` prefix guarantees the full test directory is down the list.
-FULLTESTDIR_FULL_PATH=$(readlink -e .)/$FULLTESTDIR
+# WARNING: FULL_TEST_DIR must be resolved from the current working directory.
```

Comment:
Note: This shell script should probably check from within itself whether it's on Mac or no.